modules/h2/src/main/java/org/h2/table/Table.java

```diff
@@ -1342,4 +1342,15 @@ else if (b == ValueNull.INSTANCE)
 
         return (comp > 0) == bigger ? a : b;
     }
+
+    public Column[] getVisibleColums() {
+        List<Column> visibleCols = new ArrayList<>(columns.length);
```

Comment:
may be stream operations?

Reply:
The source version of the h2 module is 1.7.
Streams not supported.